packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -175,16 +176,12 @@ $countrySelector-flag-height : 16px;
     @include media('>=mid') {
         display: none;
     }
-}
-
-.c-nav-popoverList-go-back-icon {
-    transform: rotate(180deg);
-    width: 28px;
-    height: 28px;
-    display: inline-block;
 
-    use {
-        fill: $countrySelector-icon-color;
+    .c-nav-popoverList-go-back-icon {
```

Comment:
What's the reason for this now being nested within `.c-nav-popoverList-header-button`?

Reply:
I moved it at first to overwrite f-button component styles but later decided not to use f-button for this button. But the class if very specific and won't be used outside of `c-nav-popoverList-header-button` so nesting shouldn't be a problem. But I can move it back if you like for consistency

Reply:
I did this to overwrite f-button width and height styles. We will fix it in f-button component later because at the moment it is more specific than class in the consuming app.